docs/release-notes/NuGet-6.0.md

```diff
@@ -13,14 +13,15 @@ NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**6.0.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.0](https://visualstudio.microsoft.com/downloads/) | [6.0.0](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+| [**6.0.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.0.0](https://visualstudio.microsoft.com/downloads/) | [6.0.0](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+| **6.0.1** | [Visual Studio 2022 version 17.0.2](https://visualstudio.microsoft.com/downloads/) | N/A |
```

Comment:
Do we need to update 1 more time with `https://github.com/NuGet/Home/issues/11367` and `sdk ` links later?

Reply:
Issue 11367 is already listed in the "what's new in 6.0.1" section.

There were no changes to any code shipping in the .NET SDK, so NuGet 6.0.1 is not in the .NET SDK, hence why it says "N/A" rather than having a link.  This was exactly why I added the note that Jon asked me to remove 😕